docs/tutorial.rst

```diff
@@ -869,6 +869,337 @@ of realistic size for real data.
+chromosome in the population of our choice at a specific position in the contig.
+We must also decide the time the mutation will be added, when selection will
+start and at what frequency we want our selected mutation to be at the end of
+the simulation.
+
+Let's assume the mutation appeared 1000 generations ago, it has a positive
+effect on fitness (s=0.5). Also, we want the mutation to have reached a frequency
+of at least 0.8 by the end. Next, we'll walk through the steps required to do this:
+
+.. note::
+
+    Note that because we are doing a forward-in-time simulation, you should be
+    careful with your conditioning. For example, even a strongly selected mutation
+    would not be able to reach 80% frequency in just a few generations. Since
+    this conditioning works by re-running the simulation until the condition is
+    achieved, a nearly impossible condition will result in very long run times.
+
+First, we need to define the mutation type for the selected mutation.
+So we can refer to it later, we need it's "mutation type ID"; since we've
+appended the mutation type to the end of the contig's list, that ID
+will be the length of the contig's list of mutation types:
+
+.. code-block:: python
+
+    contig.mutation_types.append(
+        stdpopsim.ext.MutationType(
+            distribution_type="f",
+            dominance_coeff=1.0,
+            distribution_args=[0.5],
+            convert_to_substitution=False,
+        )
+    )
+    mut_id = len(contig.mutation_types) - 1
+
+Next, we will set up the "extended events" which will modify the demography.
+The first extended event is placing of the selected mutation,
+which will occur in a random individual from the first population (id 0),
+in the middle of the contig, 1000 generations ago.
+We specify ``save=True`` to ``stdpopsim.ext.DrawMutation``
+so that the simulation can restart from that point if the mutation is lost.
+
+.. code-block:: python
+
+    coordinate = round(contig.recombination_map.sequence_length / 2)
+    T_mut = 1000
+    extended_events = [
+        stdpopsim.ext.DrawMutation(
+            time=T_mut,
+            mutation_type_id=mut_id,
+            population_id=0,
+            coordinate=coordinate,
+            save=True,
+        )
+    ]
+
+Next,we condition on the mutation not being lost.
+Since in the next step we condition on the mutation being at 80% frequency at
+the end, this is redundant, but it allows the simulation to immediately restart
+from any generation in which the mutation is lost, rather than waiting until the end.
+Note that this conditioning must start one
+generation after the mutation is placed, for which we use
+``stdpopsim.ext.GenerationAfter(T_mut)``.
+We cannot simply specify ``T_mut - 1`` if rescaling is present,
+otherwise the conditioning would start
+at the same generation when the mutation is placed.
+
+.. code-block:: python
+
+    extended_events.append(
+        [
+            stdpopsim.ext.ConditionOnAlleleFrequency(
```

Comment:
This doesn't look right. I think this should be appending an element, not a list containing one element.